### PR TITLE
Added a vuejs + webpack example for those deploying a VueJS app built…

### PR DESCRIPTION
**Pull request by benmccallum**
… with `vue init webpack` using the https://github.com/vuejs-templates/webpack template.

### CODE REVIEW

**Comment by benmccallum** [comment]
Hopefully this might help someone.

**On another note...**
I also spent a couple of days banging my head over how to trigger an Azure CDN endpoint purge from deploy.cmd in an elegant way. It took me forever to figure out the authentication part. The documentation is convoluted and confusing. [Here](https://docs.microsoft.com/en-au/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service#request-an-access-token) talks about using your Azure AD registerd application (Service Principle's) App ID URI and calling the endpoint (https://login.microsoftonline.com/<tenant id>/oauth2/token) but I guess that's for other services/uses. 

Finally your [blog post](http://blog.davidebbo.com/2015/12/calling-arm-using-plain-rest.html) and [sample repo](https://github.com/davidebbo/AzureWebsitesSamples/blob/master/HttpClientSample/AuthenticationHelpers.cs) using the API directly put me in touch with the right combo of oauth base url (https://login.windows.net/<tenant id>/oauth2/token) and "resource" parameter for the ARM API (https://management.core.windows.net/). 

At one point I was trying ARMClient.exe but although it worked locally it didn't work in the kudu context (couldn't save token/s to user data). 

Long story short, I ended up with [this](https://gist.github.com/benmccallum/9e2ab659282b311142fcb0022e4767c3) in my deploy.cmd. It's not perfect (is saving the access_token to the filesystem temporarily a vulnerability? suggestions?) but maybe we could put it in this repo somewhere? Could help some folks. Let me know how/where and I can do another PR.

**Review by davidebbo — APPROVED** [review]

**Comment by davidebbo** [comment]
Thanks for the sample.

For the CDN part, it's not really related to App Service, and more about calling an ARM API from curl in general. One option to put it out there is a self-answered StackOverflow question.

**Comment by benmccallum** [comment]
Good call, will do. Cheers.
